db/postgres/index.js

```diff
@@ -0,0 +1,34 @@
+const { Pool } = require('pg');
+
+const pg = new Pool({
+  user: 'keewooklee',
+  password: '',
+  port: 5432,
+  host: 'localhost',
+  database: 'reviewsdc',
+});
+
+pg.connect((err) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log('connected to DB(postgres)');
+  }
+});
+
+pg.query(`CREATE TABLE IF NOT EXISTS reviews (
+  id SERIAL PRIMARY KEY,
+  product_id int,
+  rating int,
+  summary VARCHAR,
```

Comment:
From the BRD, it states: Review Summary - Reviews submitted will have a one sentence summary. This single sentence will be capped at 60 characters.  Consider adding a max length for the character limit!